_posts/2022-12-21-trino-summit-2022-astronomer-recap.md

```diff
@@ -0,0 +1,93 @@
+These tasks may depend on each other and have a complex workflow. To orchestrate
+this complexity, this talk covers using Airflow's new Trino integration to
+run Trino batch queries to solve (almost) all your data problems.
+
+<!--more-->
+
+{% youtube xKDN7RUJ5i4 %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@Astronomer.pdf">
+  Check out the slides!
+</a>
+
+## Recap
+
+This talk covers a fictional trading scenario that initially follows a
+traditional warehousing approach to storing data. This architecture has data
+sources that are queried and submitted as raw data into a centralized warehouse.
+Within the warehouse itself, the raw data is transformed into data ready to be
+consumed.
+
+This model enforces centralization which generally involves one team to run the
+platform and build the integration between producers and consumers. This team
+focuses on the aspects of the data platform which separates them further from
+the business use case. As source databases evolve, the central data team must
+keep up with these changes. As the teams that rely on the data infrastructure
+grow, this team commonly becomes a bottleneck.
+
+Trino allows you to move the queries as close as possible to the federated data
+sources. This removes the labor-intensive process of moving data into stages
+before ingesting it into a central warehouse. This doesn't mean that data
+movement is no longer a necessity, but it does mean that the necessity shifts
+from an availability concern to a performance and scalability concern. 
+
+Without investing into more resources, your data professionals are able to work
+closely with producers and stakeholders with a shared understanding of the
+domain. This increases data literacy and data availability throughout your
+organization.
+
+Trino is not only for fast adhoc analytics with a human in the loop, but now 
+provides a fault-tolerant execution mode that enables it to run resource
+intensive batch jobs. This, paired with the federation capabilities, make Trino
+able to ingest any data that can be represented in a tabular format. Users can
+implement user-defined functions and run transformations using SQL without
+involving intermediate systems.
+
+To run Trino batch queries at scale requires building complex interdependencies
+between different tasks and often needs monitoring if there are any failures
+that occur. It also needs reactive automation to handle those instances. Apache
+Airflow is an open-source platform for developing, scheduling, and monitoring
+batch-oriented workflows on systems like Trino.
+
+The talk then covers:
+
+* Basic job creation
+* Sharded job creation that splits the large jobs into smaller tasks that
+  can easily be retried
+* Dynamic task mapping which splits jobs into smaller tasks based on data
+  observed at runtime
+* Data aware scheduling can schedule tasks based on interdependencies between
+  datasets
```

Comment:
It feels very off to have the majority of this post be an advertisement for FTE and Trino capabilities, then summarize all the Airflow-specific talk content in a bulleted list with no practical information. Needs a rewrite, even if it's just one or two paragraphs with the high level Airflow concepts.